From 995fd0995424fe7843cd73d05a9c82a9b06daee6 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Mon, 12 May 2025 15:05:46 +0200
Subject: [PATCH 7/8] man: update raid man

Mention repair of transiently lost devices.

(cherry picked from commit 22364ce9b68b3a2e03819bfd8fdc569df584b7e2)
---
 man/lvmraid.7_main | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/man/lvmraid.7_main b/man/lvmraid.7_main
index 5b429a410..db8009d46 100644
--- a/man/lvmraid.7_main
+++ b/man/lvmraid.7_main
@@ -434,7 +434,7 @@ found, run:
 # lvs -o name,raid_mismatch_count
 .EE
 .P
-Also, if mismatches were found, the 
+Also, if mismatches were found, the
 .BR lvs (8)
 attr field will display the letter
 "m" (mismatch) in the 9th position, e.g.
@@ -862,6 +862,21 @@ those PVs can be reconstructed with:
 The rebuild option can be repeated with different PVs to replace the data
 on multiple PVs.
 .
+.SS Reactivating arrays after temporary device loss
+
+When a RAID array loses a critical number of devices,
+causing it to lose its ability to function reliably,
+the array will stop and require repair.
+
+Initiate repair process with this command:
+.P
+.B lvconvert --repair
+.I LV
+.P
+If the previously unavailable devices become accessible again,
+this repair process will update their metadata and
+the RAID array can be reactivated.
+.
 .SH DATA INTEGRITY
 .
 The device mapper integrity target can be used in combination with RAID
-- 
2.49.0

